### ISSUE TITLE
RFIDTags:126: error: invalid conversion from 'byte {aka unsigned char}' to 'MFRC522::StatusCode' [-fpermissive]

### ISSUE THREAD

**Issue by abdulazizalmass** [smerkousdavid/InternetRFIDTags#3]
Bought Another working Genuino 
Ethernet is library is working 
RFID is working

Code is not

IDE stops at 
`Serial.println(mfrc522.GetStatusCodeName(status));
`

Error Message

```

WARNING: Category '' in library UIPEthernet is not valid. Setting to 'Uncategorized'
/Users/Omar/Downloads/arduino/smerkousdavid-InternetRFIDTags-46a7f8a/RFIDTags/RFIDTags.ino: In function 'void loop()':
RFIDTags:116: error: invalid conversion from 'byte {aka unsigned char}' to 'MFRC522::StatusCode' [-fpermissive]
     Serial.println(mfrc522.GetStatusCodeName(status));
                                                    ^
In file included from /Users/Omar/Downloads/arduino/smerkousdavid-InternetRFIDTags-46a7f8a/RFIDTags/RFIDTags.ino:18:0:
/Users/Omar/Documents/Arduino/libraries/MFRC522/MFRC522.h:384:36: error:   initializing argument 1 of 'static const __FlashStringHelper* MFRC522::GetStatusCodeName(MFRC522::StatusCode)' [-fpermissive]
  static const __FlashStringHelper *GetStatusCodeName(StatusCode code);
                                    ^
RFIDTags:126: error: invalid conversion from 'byte {aka unsigned char}' to 'MFRC522::StatusCode' [-fpermissive]
     Serial.println(mfrc522.GetStatusCodeName(status));
                                                    ^
In file included from /Users/Omar/Downloads/arduino/smerkousdavid-InternetRFIDTags-46a7f8a/RFIDTags/RFIDTags.ino:18:0:
/Users/Omar/Documents/Arduino/libraries/MFRC522/MFRC522.h:384:36: error:   initializing argument 1 of 'static const __FlashStringHelper* MFRC522::GetStatusCodeName(MFRC522::StatusCode)' [-fpermissive]
  static const __FlashStringHelper *GetStatusCodeName(StatusCode code);
                                    ^
/Users/Omar/Downloads/arduino/smerkousdavid-InternetRFIDTags-46a7f8a/RFIDTags/RFIDTags.ino: At global scope:
RFIDTags:225: error: expected declaration before '}' token
 }
 ^
Multiple libraries were found for "Ethernet.h"
 Used: /Users/Omar/Documents/Arduino/libraries/Ethernet
 Not used: /Applications/Arduino.app/Contents/Java/libraries/Ethernet
exit status 1
invalid conversion from 'byte {aka unsigned char}' to 'MFRC522::StatusCode' [-fpermissive]
```

**Comment by smerkousdavid** [comment]
They must have updated it, anything involving Serial print you can just delete as it's for debugging... So you can comment out those lines (Or I will in the repo).

**Comment by oliverwarne** [comment]
Resolved with commit 90b6015001027a47d91f9f4308bb4c26bd9e9140.
